PelotonEppSdk/Models/EventItem.cs

```diff
@@ -78,6 +82,28 @@ internal class event_item
                     CustomFields = customFields
                 };
             }
+
+            public static explicit operator event_item(EventItem ei)
+            {
+                if (ei == null) return null;
+
+                ICollection<event_custom_field> customFields = null;
```

Comment:
Yes, almost -- I think if I had one more question mark to your code snippet, then it will work.

`var customFields = ei?.CustomFields?.Select(cf=> (event_custom_field) cf).ToList()`